src/abstract-interpretation/data-frame/absint-info.ts

```diff
@@ -16,6 +16,10 @@ interface DataFrameInfoBase {
 	domain?: DataFrameStateDomain
 }
 
+export interface DataFrameUnassignedInfo {
+	type: 'unassigned'
+}
+
 export interface DataFrameAssignmentInfo {
 	type:       'assignment',
```

Comment:
In general a short overview for the scenarios in which the respective infos apply